core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/SbomUtils.java

```diff
@@ -412,6 +429,26 @@ private static void addLicenseEvidence(Component component, List<LicenseInfo> li
         component.setEvidence(evidence);
     }
 
+    public static List<Hash> getDistroHashesFromAnalyzedArtifact(AnalyzedArtifact analyzedArtifact) {
+        List<Hash> hashes = new ArrayList<>();
+
+        if (analyzedArtifact.getDistribution() == null) {
+            return hashes;
+        }
+
+        // Use our pre-defined Algo to getter mapper here (DIST_HASH_DEFINTIONS)
+        for (HashAlgorithmMapping m : DIST_HASH_DEFINITIONS) {
+
+            // eg. Call analyzedArtifact.getDistribution.getSha256 and return a cdx Hash of type SHA_256
+            String hashValue = (String) m.getter.apply(analyzedArtifact.getDistribution());
+
+            if (Objects.nonNull(hashValue) && !hashValue.isEmpty()) {
+                hashes.add(new Hash(m.algorithm, hashValue));
+            }
```

Comment:
I don't think that checking `!isEmpty()` is necessary since CycloneDxGenerateOperationCommand.java:174 does not do it and only checks for `null`.